gnd/src/main/java/com/google/android/gnd/ui/common/Navigator.java

```diff
@@ -110,7 +110,9 @@ public void showHomeScreen(int currentNavDestinationId) {
         navigate(SignInFragmentDirections.proceedToHomeScreen());
         break;
       default:
-        throw new IllegalArgumentException(currentNavDestinationId + " id not found");
+        // Do nothing, probably a config change
```

Comment:
What happens if we add a one-sided direction to the home screen and use that here?

Also, could we log an error (Timber.e) here to make this easier to debug?

Reply:
> What happens if we add a one-sided direction to the home screen and use that here?

During config change, the navigation doesn't happen via Navigator. So the destination id is still unset.

> Also, could we log an error (Timber.e) here to make this easier to debug?

Done